web-quiz-app/src/pages/Login.jsx

```diff
@@ -7,51 +7,85 @@ class Login extends React.Component {
         this.state = {
             login: false,
             name: "",
+            username: "",
+            password: "",
             isInstructor: true
         }
     }
+
     nameUpdate = (currentName) => {
         this.setState({
             name: currentName
         });
         console.log(this.state.name);
     }
+
+    usernameUpdate = (currentUsername) => {
+        this.setState({
+            username: currentUsername
+        });
+        console.log(this.state.username);
+    }
+
+    passwordUpdate = (currentPassword) => {
+        this.setState({
+            password: currentPassword
+        });
+        console.log(this.state.password);
+    }
+
     isInstructorUpdate = (isInstr) => {
         this.setState({
             isInstructor: isInstr
         });
     }
 
-    createUser(name) {
+    createUser(name, username, password) {

```

Comment:
I think there will likely be some slight conflict with this PR and the update Login page PR, but I expect this one is more complex, so I'll update my PR after you merge in this one.